### PR TITLE
The Next Web

### PR DESCRIPTION
**Pull request by sanadwj**
## What this PR does 

-Make a responsive design for The Next Web

### CODE REVIEW

File: index.html
Comment (kitpao) [inline index.html:70]: - [x] On **mobile** and **tablet** notice that this element disappears, please hide it in said versions
image

File: styles.css
Comment (kitpao) [inline styles.css]: - [x] **[OPTIONAL]** You can emulate the height of the original, by using vh units on this height, so that it is relative to the screen height.
image

File: styles.css
Comment (kitpao) [inline styles.css:911]: - [x] **[OPTIONAL]** This applies to all your images: To avoid stretched images, you can use `object-fit: cover;` and they will keep their original proportions

File: styles.css
Comment (kitpao) [inline styles.css]: - [x] On **mobile** and **tablet** please notice that this margin is giving unnecessary space between sections
image

File: index.html
Comment (kitpao) [inline index.html]: - [x] On **mobile** notice the responsive behavior in these links, in the original they are in 1 column and the pipeline (|) dissapears
image

File: index.html
Comment (kitpao) [inline index.html:273]: * This applies for all Apps, Gear, Tech, Creative, Podium, Insights, Launch, and Distract sections

- [x] On **mobile** notice that the image is larger in height, please fix it
- [x] On **mobile**, **tablet** and **desktop** your orange "latest" section is not showing, please display it so it looks like the original

image

File: index.html
Comment (kitpao) [inline index.html:37]: - [x] In desktop, the logo doesn't appear, please hide it with media queries
image

File: styles.css
Comment (kitpao) [inline styles.css:228]: - [x] On **desktop** your sections .latest-news, .latest-funding and .tech-section are not aligned correctly, you can fix this, simply by increasing the % width of said sections
image

**Review by kitpao — CHANGES_REQUESTED** [review]
# 👋 Hello @Miguelus373 and @sanadwj 

You have nice commit messages, they are descriptive and useful ✔️ Your code is nicely organized 👍 and most importantly about your project in general, you did it amazingly ⭐ especially for the first time submitting it 👏 You are just missing some details in order for it to be approved, please check below and in the inline comments 🔽 

**_Note:_** All the **[OPTIONAL]** are **not required** to approve your project but are personal suggestions to improve your coding experience

### Implementation details

- [ ] As the requirements in pathwright state, you should use the **mobile-first** approach, that means that the default is the small screen and no media queries are used for this size, next you should state the media queries for tablet `@media screen and (min-width: 768px)` and lastly the media queries for desktop `@media screen and (min-width: 1024px)`. [You did desktop-first and established max-widths for your media queries]. Please correct this. 
image

- [ ] Although you did put a description in your pull request message, remember that it should have a summary of the things you did (write all the features you added), like in the example
image


### ♻️ Status: Your project requires some additional changes ♻️
➡️ Please request another review once you are ready 📝 

___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

File: styles.css
Comment (sanadwj) [reply styles.css:228]: Hello @kitpao thanks for the review, We fixed the requested changes and the breakpoint sizes and I reached to @Bola and explain everything and I waiting for the reply.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Changes required 🛑 

## Impressive progress so far 💪! Thanks for making all the changes 👏.

### However, we need to work on some changes before we can merge.

- In mobile view, the `.bottom-footer` is misaligned 👇 

  Annotation 2020-07-30 230856

  ...and the social icons in the footer starts overflowing 👇 

  Annotation 2020-07-30 230937

  Please fix these.


- As for the mobile-first approach, you only need to move the styles for the larger screen into media query and make the small screen default styles, that is, without media query. And also, use min-width for your media queries. 

Your project looks absolutely fantastic. Just make these changes and apply for another code review.

Happy coding 👨‍💻 

*Need help? Shoot me a message on Slack - [@Ghosh](https://app.slack.com/client/T47CT8XPG/DRDM6LAQN/user_profile/URB1MHXK6)*
<hr>

**Tirthajyoti Ghosh**

[GitHub](https://github.com/tirthajyoti-ghosh) | [LinkedIn](https://www.linkedin.com/in/tirthajyoti-ghosh/) | [Twitter](https://twitter.com/terrific_ghosh)

**Comment by sanadwj** [comment]
Hello @Ghosh  thanks for the review, We fixed the requested changes.

Screenshot from 2020-07-30 21-32-02

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :clap: 

You did a very good job on the changes requested above, especially with using a mobile-first approach. :1st_place_medal: 

Also, the page looks awesome :100: 

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**
